src/sentry/models/groupassignee.py

```diff
@@ -75,7 +79,27 @@ def assign(self, group, assigned_to, acting_user=None):
             )
             activity.send_notification()
 
+        # sync Sentry assignee to external issues
+        if isinstance(assigned_to, User) and features.has(
```

Comment:
I'd check against `assignee_type == 'user'` since we've already done that above.